### PR TITLE
Top tabs android2

### PR DESCRIPTION
**Pull request by guyca**

### CODE REVIEW

File: lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/toptabs/TopTabsController.java [hunk 1]
```diff
@@ -22,12 +23,17 @@ public TopTabsController(Activity activity, String id, List<ViewController> tabs
...
 
     @NonNull
     @Override
     public Collection<ViewController> getChildControllers() {
         return tabs;
     }
+
+    @Override
+    protected boolean isViewShown() {
+        return super.isViewShown();
```
Comment (DanielZlotin) [inline lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/toptabs/TopTabsController.java]: is this correct?

File: lib/android/app/src/main/java/com/reactnativenavigation/views/Container.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.reactnativenavigation.views;
+
+import com.reactnativenavigation.viewcontrollers.ContainerViewController;
+
+public interface Container extends ContainerViewController.IReactView {
+    TopBar getTopBar();
```
Comment (DanielZlotin) [inline lib/android/app/src/main/java/com/reactnativenavigation/views/Container.java]: add TODO with description to refactor this breakage of SRP

Comment (DanielZlotin) [reply lib/android/app/src/main/java/com/reactnativenavigation/views/Container.java]: or open issue

File: lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/toptabs/TopTabsController.java (hunk 1 above)
Comment (guyca) [reply lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/toptabs/TopTabsController.java]: Not entirely,
I think `ParentViewController` should check if all child controllers are shown, wdyt?

Comment (DanielZlotin) [reply lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/toptabs/TopTabsController.java]: hmm.. what about sideMenu controller and other composable parents? not all child controllers are shown yet the parent is show..

File: lib/android/app/src/main/java/com/reactnativenavigation/views/Container.java (hunk 2 above)
Comment (guyca) [reply lib/android/app/src/main/java/com/reactnativenavigation/views/Container.java]: Good point. #2348

File: lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/toptabs/TopTabsController.java (hunk 1 above)
Comment (guyca) [reply lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/toptabs/TopTabsController.java]: Will be addressed in #1083

**Pull request merged**